src/internal/frame.hpp

```diff
@@ -46,17 +55,18 @@ class Frame
    *  \retval true writing completed.
    *  \retval false writing disrupted by timeout.
    */
-  bool
-  writeTo(Print& os, int timeout = 10000);
+  bool writeTo(Print& os, int timeout = 10000);
```

Comment:
Do not change code style.
I use the NDN style: https://named-data.net/doc/ndn-cxx/current/code-style.html

Reply:
Fixed with commit -a -amend